docs/articles/lwc/localdev.md

```diff
@@ -0,0 +1,232 @@
+
+```sh
+git clone git@github.com:trailheadapps/lwc-recipes.git
+```
+4. If you're not in the project's root directory already, `cd` into it. The root of the directory has the file `package.json` in it. 
+
+```sh
+cd lwc-recipes
+```
+
+5. Add the `.localdevserver` folder in your SFDX project to your `.gitignore` file. Do not modify files inside of this folder.
+
+6. Authorize a Developer Hub (Dev Hub) by following the steps in [Enable Dev Hub In Your Org](https://developer.salesforce.com/docs/atlas.en-us.sfdx_setup.meta/sfdx_setup/sfdx_setup_enable_devhub.htm) in the *Salesforce DX Developer Guide*. A Dev Hub is the main Salesforce org that you and your team use to create and manage your scratch orgs, temporary environments for developing on the Salesforce platform. You need the Dev Hub to create a scratch org in a later step.
+
+7. Following the instructions in the *Salesforce DX Developer Guide*, log in using your Dev Hub credentials. Running the following command spawns a login window in your browser.
+
+```sh
+sfdx force:auth:web:login -d -a <myhuborg>
+```
+
+8. In local development, requests to Lightning Data Service and Apex go to scratch orgs, similar to how they go to your production org. To create a scratch org, run this command from the command line.
+
+```sh
+sfdx force:org:create -s -f config/project-scratch-def.json -a "LWC"
+```
+
+“LWC” is an alias for the scratch org that you can use in other Salesforce CLI commands.
+
+To create a scratch org, specify a scratch org definition file. This example uses the scratch org definition file, project-scratch-def.json that is included in `lwc-recipes`. For other projects, create your own. For more information, see the instructions for [Create Scratch Orgs](https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_scratch_orgs_create.htm) in the *Salesforce DX Developer Guide*.
+
+9. Start the server.
+
+```sh
+sfdx force:lightning:lwc:start
+```
+
+For more information on this command, view Help for the Local Development Server by running `sfdx force:lightning:lwc:start --help`.
+
+10. View the server at [http://localhost:3333/](http://localhost:3333/). 
+
+## Troubleshooting
+
+```sh-session
+$ sfdx force:lightning:lwc:start
+Starting LWC Local Development.
+    Dev Hub Org: mydevhub
+    Scratch Org: undefined - We can't find an active scratch org for this Dev Hub. Create one by following the steps in Create Scratch Orgs in the Salesforce DX Developer Guide (https://sfdc.co/cuuVX4) or the Local Development Server Getting Started.
+```
+
+If you see this error, make sure that you authenticate to your Dev Hub and create a scratch org. 
+
+## Configuration for Projects (Optional)
+
+SFDX automatically configures your project out of the box, but if you want to provide additional information for the server, add a localdevserver.config.json file at the base of your project. 
```

Comment:
"provide additional information" may imply these values need to be specified? In an sfdx project specifying any of these values will always be considered overrides rather than additional information... Could we say, "but if you need to override configuration for the server, add a..."

Reply:
done